odoo/cli/shell.py

```diff
@@ -1,4 +1,5 @@
-# Part of Odoo. See LICENSE file for full copyright and licensing details.
+# ruff: noqa: PLC0415
```

Comment:
Avoid adding this globally, the only place it makes sense to have global ruff noqa are some init files.

Reply:
Ah, mkay, will also fix the other PR